integration-test/test.sh

```diff
@@ -276,6 +279,7 @@ wait_for_docker() {
 			log "Authenticated to docker engine at $addr."
 			# Check if docker.options in public.json took effect
 			local docker_info_out="$(eval $docker_env $docker_cmd 2>&1)"
+            echo "DEBUG: $docker_info_out"
```

Comment:
There was a log method, I should have used that instead.

There were other whitespace issues, so I cleaned those up too.